### PR TITLE
SXG: mention support from other Chromium browsers

### LINKED ISSUE #2452: content: Fix "cross-browser support" section for signed exchanges
### What page(s) need to be updated?

https://web.dev/how-to-set-up-signed-http-exchanges/

### Why is this update needed?

The "cross-browser support" section says that Chrome is the only supporting browser. However, some other browsers based on the Chromium rendering engine also render SXG. I would file a PR, but I'm not sure how you'd prefer this be worded.

### What's the deadline?

No deadline.

### PR DESCRIPTION
**Pull request by twifkak**
Google Chrome, Samsung Internet, and Microsoft Edge, in particular.

Fixes #2452.

### CODE REVIEW

**Comment by twifkak** [comment]
cc @kumagi @jpmedley

File: src/site/content/en/blog/how-to-set-up-signed-http-exchanges/index.md [hunk 1]
```diff
@@ -16,8 +16,10 @@ tags:
 
 ## Cross-browser support
 
-Chrome is currently the only browser that supports SXG.
-See the Consensus & Standardization section of [Origin-Signed HTTP Exchanges](https://www.chromestatus.com/feature/5745285984681984) for more up-to-date information.
+Several Chromium-based browsers supports SXG, including Google Chrome, Samsung
+Internet, and Microsoft Edge. See the Consensus & Standardization section of
```
Comment (jpmedley) [inline src/site/content/en/blog/how-to-set-up-signed-http-exchanges/index.md:20]:
```suggestion
Internet, and Microsoft Edge. See the Consensus and Standardization section of
```

File: src/site/content/en/blog/how-to-set-up-signed-http-exchanges/index.md [hunk 2]
```diff
@@ -16,8 +16,10 @@ tags:
 
 ## Cross-browser support
 
-Chrome is currently the only browser that supports SXG.
-See the Consensus & Standardization section of [Origin-Signed HTTP Exchanges](https://www.chromestatus.com/feature/5745285984681984) for more up-to-date information.
+Several Chromium-based browsers supports SXG, including Google Chrome, Samsung
```
Comment (kaycebasques) [inline src/site/content/en/blog/how-to-set-up-signed-http-exchanges/index.md:19]:
```suggestion
Several Chromium-based browser supports SXG, including Google Chrome, Samsung
```

**Comment by twifkak** [comment]
Thanks!

**Pull request merged**